Longdesc is an attribute 258

chaals opened the pull request:
fix #258

Ready for review

Commit:
Merge remote-tracking branch 'w3c/master'

Commit:
Merge remote-tracking branch 'w3c/master'

Commit:
Add longdesc as a valid attribute for img

fix #258

Commit:
don't try linking in IDL

Travis doesn't like that

arronei commented:
What about longdesc on frames and iframes? All browsers support it on the `<frame> and <iframe>` elements

LJWatson commented:
It looks as though the <a href="https://www.w3.org/TR/html-longdesc/">Image Description extension</a> limits longdesc to the `<img>` element, extending the HTMLImageElement interface. If browsers currently support longdesc on `<frame>` and `<iframe>`, then that sounds like a "match reality" issue against the Image Description extension.

chaals commented:
What Léonie said. It was removed from HTML 5 drafts, and after literally years of argument, we got an extension spec. If it works on frame / frameset, we can deal with that in HTML 5.2 as part of matching reality.

adanilo commented on sections/semantics-embedded-content.include:
```diff
@@ -2164,6 +2167,9 @@
   The <dfn attribute for="HTMLImageElement"><code>isMap</code></dfn> IDL attribute must <a>reflect</a>
   the <code>ismap</code> content attribute.
 
+  The <dfn attribute for="HTMLImageElement"><{img/longDesc}> IDL attribute must <a>reflect</a>
```
Travis is correct, had to change this for the links to build correctly.

adanilo commented on single-page.bs:
```diff
@@ -741,6 +741,13 @@ urlPrefix: https://www.w3.org/TR/workers/#; spec: WORKERS;
+
+url: https://www.w3.org/TR/html-longdesc/#widl-description-longdesc; type: attribute; for: img;
+    text: longDesc
+url: https://www.w3.org/TR/html-longdesc/; type: element-attr; for: img;
```
This creates a bad link, using '#https://' etc. relative to the current spec. as parent, change to urlPrefix corrects it.

adanilo commented on sections/semantics-embedded-content.include:
```diff
@@ -597,6 +598,7 @@
           attribute DOMString sizes;
           attribute DOMString? crossOrigin;
           attribute DOMString useMap;
+          attribute DOMString longDesc; // defined in [[!LONGDESC]]
```
The [[!LONGDESC]] reference here doesn't link to anything after bikeshed runs, probably cause it's in a comment in the IDL code. Not sure if this can be fixed.

Commit:
Fix linking errors

which means some editorial shuffling too. No substantive change
compared to the previous commit

chaals commented:
Ready for review again.
